doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -399,13 +400,18 @@ Requests to the environment
 
    Displays the full name of objects from Coq's various qualified namespaces
    such as terms, modules and Ltac, thereby showing the module they are defined
-   in.  It also displays notation definitions.
+   in and the location inside the file.  It also displays notation definitions.
 
    Note that objects are reported only when the module containing them has
    been loaded, such as through a :cmd:`Require` command.  Notation definitions
    are reported only when the containing module has been imported
    (e.g. with :cmd:`Require Import` or :cmd:`Import`).
 
+   Also note that the filename reported in the location is relative to
+   the ``coqc`` call used to generate it; in order to jump to the
+   concrete file, the original project build info is needed by the
+   user inteface.
```

Comment:
```suggestion
   Also note that the filename reported in the location is relative to
   the filepath ``coqc`` process used to compile it; the original project
   build info is needed in order to the definition in a file.
```

Please be specific on how to jump to a symbol, maybe add an example or two of how it works.  This wording leaves it up to the user to guess how it works.  It seems lame to do throw this off on the user interface.  Probably you should be storing/using something like the logical name of the compiled file, then looking in the LoadPath.  This should be done on the Coq side rather than asking each IDE to write their own (which also makes it harder to change the mechanism if need be).

No need to mention "jump to" or "user interface", this is just a command giving info.

You should say that the locations can become incorrect if you edit the file after it's been compiled.  That makes the feature somewhat less valuable.  Handling that case is possible, but more work.  Also, this doesn't help you find definitions in incomplete scripts, e.g. one you're in the middle of writing.